.travis.yml

```diff
@@ -8,7 +8,7 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
+    - CONDA_R=3.3.1
```

Comment:
Thanks @bgruening. I am OK with either decision. (Actually 3.3.2 is fine for my limited R needs.)

Still... This PR is not how we should do it. Sorry @vinjana.